addons/website_slides_survey/static/src/js/public/components/slide_upload_dialog/slide_upload_category.xml

```diff
@@ -25,6 +25,7 @@
                         </DropdownItem>
                     </t>
                 </SelectMenu>
+                <span t-if="state.choices.isError" class="text-danger">Please select a certification.</span>
```

Comment:
So the `required="true"` flag on the SelectMenu component is actually useless right?

If so, would remove it to avoid any confusions.

Reply:
@awa-odoo 

The "required" attribute is necessary because if it's removed, it allows us to deselect the certificate, causing a client error: "Cannot read properties of undefined (reading 'label')."